DotNet/Census/Application/Repositories/Scheduling/CensusSchedulingRepository.cs

```diff
@@ -107,24 +119,16 @@ public void GetAllJobs(IScheduler scheduler)
                 IReadOnlyCollection<ITrigger> triggers = scheduler.GetTriggersOfJob(jobKey).Result;
                 foreach (ITrigger trigger in triggers)
                 {
-                    Console.WriteLine(group);
-                    Console.WriteLine(jobKey.Name);
-                    Console.WriteLine(detail.Description);
-                    Console.WriteLine(trigger.Key.Name);
-                    Console.WriteLine(trigger.Key.Group);
-                    Console.WriteLine(trigger.GetType().Name);
-                    Console.WriteLine(scheduler.GetTriggerState(trigger.Key));
-                    DateTimeOffset? nextFireTime = trigger.GetNextFireTimeUtc();
-                    if (nextFireTime.HasValue)
-                    {
-                        Console.WriteLine(nextFireTime.Value.LocalDateTime.ToString());
-                    }
-
-                    DateTimeOffset? previousFireTime = trigger.GetPreviousFireTimeUtc();
-                    if (previousFireTime.HasValue)
-                    {
-                        Console.WriteLine(previousFireTime.Value.LocalDateTime.ToString());
-                    }
+                    _logger.LogInformation("Group: {Group}, Job: {JobName}, Description: {Description}, Trigger: {TriggerName}, Group: {TriggerGroup}, Type: {TriggerType}, State: {TriggerState}, NextFire: {NextFireTime}, PreviousFire: {PreviousFireTime}",
```

Comment:
Consider making this a DEBUG log instead of Info. In a `GetXXX()` operation like this, I believe this _could_ be noisy and we might only want to see this when DEBUG logging is turned on (which should be the default for lower environments, but not in upper environments). And now that I'm thinking about this, I think we could use some consistency in how we approach INFO vs. WARN vs. ERROR logs across the board; so I'm drafting some starting guidance for this that I'll share separately.